.github/workflows/main.yml

```diff
@@ -8,7 +8,7 @@ env:
 jobs:
   lint:
     name: Code Style
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
Was getting blocked by a brownout since [18.04 is being deprecated](https://github.com/actions/runner-images/issues/6002), this works though 😄